src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -29,6 +29,22 @@ mocha.describe('readings API', () => {
             mocha.describe('for quantity groups', () => {
 
                 // Add BG1 here
+                mocha.it('1 day bars for 15 + 20 minute reading intervals and quantity units with +-inf start/end time & kWh as kWh', async () =>{
```

Comment:
1) Could you please remove the comment above about adding BG1 since it is now done.
2) Could you please add BG1: at the start of the mocha.it so it is shown when the test runs.